test/metabase/test/data/vertica.clj

```diff
@@ -22,16 +21,16 @@
 
 
 (defn- db-name []
-  (or (env :mb-vertica-db)
-      "docker"))
+  (i/db-test-env-var-or-throw :vertica :db "docker"))
 
 (def ^:private db-connection-details
-  (delay {:host     (or (env :mb-vertica-host) "localhost")
+  (delay {:host     (i/db-test-env-var-or-throw :vertica :host "localhost")
+          :port     (Integer/parseInt (i/db-test-env-var-or-throw :vertica :port "5433"))
+          :user     (i/db-test-env-var :vertica :user "dbadmin")
+          :password (i/db-test-env-var :vertica :password)
           :db       (db-name)
-          :port     5433
           :timezone :America/Los_Angeles ; why?
-          :user     (or (env :mb-vertica-user) "dbadmin")
-          :password (env :mb-vertica-password)}))
+          }))
```

Comment:
oh... right. I'm not sure that `:timezone` is used or if we need it. But that's something we can tackle another time so this looks good to me